### PR TITLE
add vahid to names file

### PR DESCRIPTION
**Pull request by vsharafi**

### CODE REVIEW

File: names.txt [hunk 1]
```diff
@@ -12,3 +12,4 @@ Alireza
 Sadegh
 Lenis03
 Saida
+vahidsh
```
Comment (lenis03) [inline names.txt:15]: Hello, please just add your first name to the end of the file!

**Review by lenis03 — CHANGES_REQUESTED** [review]

**Review by AliPrj — APPROVED** [review]
Thanks 😊

**Review by AliPrj — APPROVED** [review]
Thanks

**Review by AliPrj — APPROVED** [review]
Thanks

**Review by AliPrj — APPROVED** [review]

**Pull request merged**